core/arch/arm/plat-hikey/platform_config.h

```diff
@@ -38,10 +38,10 @@
 #endif /* ARM64 */
 
 /* PL011 UART */
-#if defined(CFG_HIKEY_UART3)
-#define CONSOLE_UART_BASE	0xF7113000
-#else
+#if defined(CFG_HIKEY_UART0)
```

Comment:
Good point, so to summarize what you and I are saying:
* In build.git, use: `CFG_NW_CONSOLE_UART` and `CFG_SW_CONSOLE_UART`
* And for the real projects, use: `CFG_CONSOLE_UART`